tests/test_utils/test_chroma_store.py

```diff
@@ -0,0 +1,161 @@
+"""Test suite for testing chroma_store utility."""
+import tempfile
+
+import chromadb
+import pytest
+from chromadb.api import API
+from chromadb.api.types import Documents, EmbeddingFunction, Embeddings
+from chromadb.config import Settings
+from utils.chroma_store import ChromaStore
+
+
+@pytest.fixture
+def local_persist_api() -> API:
+    """Fixture to create a local chroma server for testing.
+
+    Returns:
+        API: Local chroma server
+    """
+    return chromadb.Client(
+        Settings(
+            chroma_api_impl="local",
+            chroma_db_impl="duckdb+parquet",
+            persist_directory=tempfile.gettempdir() + "/test_server",
```

Comment:
There's a pre-existing temporary directory fixture which could be used here? You'll need to pull the changes in `develop` into here to see it in the high-level `conftest`

Reply:
We cannot use `directory_for_testing` as it creates a new directory for every test using `local_persist_api` as input. This means we creates different instance of chromadb for each test which is not what we want.